templates/ssh/authorized_keys

```diff
@@ -43,7 +43,7 @@ ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCuYbVqE2p2NzKow9scMp4XvrIIf3GVSmV5hsI7aXbe
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDEC7d2xyWDZ0JXFNTpC5h5XWDasZCkYoc36yTfzGZc4YNmuyrEyKH4MVRWipMq8oHoQ/YoFGwv1t4dWMo8hqohAun3E2khbujl9LKxJbriM1b6BdDiR+Rn9H6TtOQPxVLj8trJmG9CIhzWjdKCQAMAeejQUQgfjOZ9uX4QMn1/SdEpUgkxuNgTdbUeWoLu+u1wNgBEyumLeAbiKdz0Ar2LDpqcjwB0JZLwA7Wvh+37MxVuPh8SOM1qcj5uZ251tsVJ3xCbiuZi+T5F2oZA7umd0SxHERuuTHYl/B48CtkU2uLtisME7NQDK7WgIyMCyc2Gn3QbAaUtNr2m/GjeqaNH michael.smith@puppetlabs.com
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCnkp83vDmBv/CO0blBmmb8pcuyBsjfX0feNvCacbqpSV5iDQC64g5TSNHwLqT5WkcHhhPbRtJa5PsNAu0vZESjHx+arjKlhVrHpQmmvLe8x5UT03WXeAC5NfHRgxgXJB3P7s0YVgL/JoZQfdpqSnifuUNTVz8/sFp8+dZtIT1edWY4r+Nu+xkKSlTLJemUx6N4ghDKzbh/4UJn4+UjXj/+cN1nnhz/FfsSYFq7VUQSFkBByPWSWBWtYmGT/tSY3wzBU1B10034LKLOr3ZnLC+i1xUECg+lfTuUmJ20XunnyY5V1OgVFjq3akPdSWlx9J+Cwl/XdjD6Su49xhurTP5X lmacchi@lauras-mbp.corp.puppetlabs.net
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCqipbVMpiJcXR02uoRccYG371qx6RvjSAvZMqYM46+SGT+mldJiIZKcV9qf0D+OzND6fPiPYWaQFhh+tg9wzjb5wj0naMoaI5X2jLBjlRbpFEI26vXB4e2QOfH4tWef6bcHjWAeOrlm7KpsgTA9S1imRdrKKEBXTupJIyyN7bxCX2az15QlAeNY19Lj03B7uiHpGBgBfdPKB7u1Qewi8CIzeNEhv5jQjH0YjT3RGrsipch7Og9GM0qWu8jDVlxDNImG/jgrOEJ74nOHnBXoIaKHdpIXrUfV8jW3tsVZdXLF046oTDNP9q+/+v23LPL5qgHnRKyNZcylkSm9O6BhXVX/TkNc+Q1n4M5jKqG+hgGqw0aH/+gGKNxbUTeREMrz50gBI3Lj3RJKg1NuslHZtPNyWqLXhA6uyo8ZvmNrHkFXirJHIREeL91WPF8V90bKNqOdfTfE7WFGMvpGf6USrEvkjApstY4ODNmopudvztKow0yqGoSj5c8bdU205ynga3sEpW5sopgCfwYyPoZC17CUkJJHR5C+vtZbeNfciF/ap4+1qySGPaWeRtgrfPDr68KtOm19zv7NzO4bK6TWFQtouSKgX8RXZT6k6KMwOanjTIRklJqL5X9qt7z2SrXIIb/Gxj8eun6zRYBH5jb9NvoR4hox1lwTzd0F87IvlA5tQ== danieldreier@daniels-mbp.corp.puppetlabs.net
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDH0ba8AmBRKhV/ekeiDfTnCNeJ4tItEyG/K6hLDASP1/8H98h7NUxVLyQG+SwSekyHjjSweePTPCLq12tZGzD0FTn2AYgL+fNiYi1uUtZHtcjIxgpvcdGa3sNWsGK00TtxwLA2vUzX7RAvFIYs2OFiHX/Vyn2Rms9Pq/FoMvGpIhS6WkuqNd1K8PDOL/LpLRRVe+j63W5s/vgDXIE3gIZSE6hUvI82M7jze0J+h4X0cSE6RpJXMjnPeXoCy65NhvRrp90omU0aWT1tQUY7fdR4nSFDxtJSOYnPVmLCAscIoGO3iYPTMsBAzBSowQERBCckeJmMJR8XmvaJ4MujiZ7wktVCq4lh19PsfOPP+RzZP9CphddtBaBXCyuDc9gUNF+v1ldP6gG+tIhlA1i0dflOTB6b8yfFLcje0jqUaUbFolkBZB4CZMhRDTEGxybXb7spkVpgltBUKcGELK9lIA1c2Bpdfg6RQeCr6IbJoTSK9z541r6cm7RBTJghvwu340TN+83tW7rh0l5GNh41gQLDlZlyQgj39353V7OmZsygaNU4gJEP8PzlooLgKUv8UQ4QBO77U5huxWUdw5vjWrdKpFjnu0CAqlliecPe0SyKFm6Js1j+MZMaY+/R7ZAY8NGjqlN7/akqsQ1Pku+RBfXfV9Py0BszrQndleAJedoWhw== britt@puppetlabs.com
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDH0ba8AmBRKhV/ekeiDfTnCNeJ4tItEyG/K6hLDASP1/8H98h7NUxVLyQG+SwSekyHjjSweePTPCLq12tZGzD0FTn2AYgL+fNiYi1uUtZHtcjIxgpvcdGa3sNWsGK00TtxwLA2vUzX7RAvFIYs2OFiHX/Vyn2Rms9Pq/FoMvGpIhS6WkuqNd1K8PDOL/LpLRRVe+j63W5s/vgDXIE3gIZSE6hUvI82M7jze0J+h4X0cSE6RpJXMjnPeXoCy65NhvRrp90omU0aWT1tQUY7fdR4nSFDxtJSOYnPVmLCAscIoGO3iYPTMsBAzBSowQERBCckeJmMJR8XmvaJ4MujiZ7wktVCq4lh19PsfOPP+RzZP9CphddtBaBXCyuDc9gUNF+v1ldP6gG+tIhlA1i0dflOTB6b8yfFLcje0jqUaUbFolkBZB4CZMhRDTEGxybXb7spkVpgltBUKcGELK9lIA1c
```

Comment:
It looks like this portion of the key is being re-added?

Reply:
I'm going to close this PR and open another to remove Britt's key.